### DISCUSSION TITLE
Thorium  android 124.0.6367.218 version keeps crashing!!

### DISCUSSION THREAD

**Discussion by RAGNODIN** [Alex313031/thorium#710] [Q&A]
I installed the recent Android version(124.0.6367.218) on my phone which is s22 ultra Android 14. I was able to use after installing but when I quitted from the app and wanted to use few moments later, app was not opening at all. I also use it in pc(linux) and no issue in there.

**Comment by vippium** [comment]
> I installed the recent Android version(124.0.6367.218) on my phone which is s22 ultra Android 14. I was able to use after installing but when I quitted from the app and wanted to use few moments later, app was not opening at all. I also use it in pc(linux) and no issue in there.

I personally recommend you to switch to any other privacy related brower, as Thorium on android is discontinued for infinite time. Try Cromite for it. OR Firefox

**Comment by gz83** [comment]
The Android version no longer crashes.
